### ISSUE TITLE
NPM maintainers?

### ISSUE THREAD

**Issue by rreusser** [scijs/contributing#18]
Is it necessary or even possible to establish some sort of npm organization in order to prevent orphaned modules? I see this: https://docs.npmjs.com/cli/owner but that sounds like we'd have to manually add a couple maintainers for every module, which would be a pain unless done programatically. Just trying to figure out how to avoid a bad situation.

**Comment by rreusser** [comment]
Of course it may be possible for someone else with repo access to just publish a new version, but I'm not 100% sure.

**Comment by Planeshifter** [comment]
`npm owner add` is not so bad. I do not think npm has organizations or anything like that (yet).

**Comment by kgryte** [comment]
Repo access is not enough. You do have to be made an `owner` of an NPM module. One way to automate the process is via git hooks or some other build process. 

I have started working on some modules to help us with these type of issues. When we have something more formalized, I will be sure to share.

**Comment by rreusser** [comment]
Cool. Then just go with the flow if I just go ahead and add someone. :smile: I'm at a point in life where I'm really sick of wasting time on ill-fated projects of one sort or another.

**Comment by rreusser** [comment]
Added @Planeshifter + @mikolalysenko to a bunch of stuff. Nothing personal, @kgryte! Don't hesitate to ask if you want to contributing to anything for which I have permissions. Just a redundancy check until a better solution is found.

**Comment by kgryte** [comment]
+1 :)

**Comment by mikolalysenko** [comment]
Yeah, I need to be more diligent about this, but it is hard to stay on top of with the npm interface.  I think @hughsk had a script which automated some of this?  Anyway as a temporary measure I'll start adding some maintainers manually.

**Comment by hughsk** [comment]
https://github.com/repo-utils/npm-org-rights does wonders :)

**Comment by hughsk** [comment]
Oh, important to note that you need iojs or `node --harmony` for that. I generally switch over with [nvm](https://github.com/creationix/nvm) and then use `node --harmony $(which npm-org-rights)`.

**Comment by rreusser** [comment]
Awesome, thanks! Yeah, just trying hard to push some of this tedious organizational stuff forward. I'll see if I can set that up.

**Comment by mikolalysenko** [comment]
Ok!  It is done.  I added all you guys to scijs using npm-org-rights.  You all should do the same!

**Comment by mikolalysenko** [comment]
The syntax for that command is:

```
npm-org-rights scijs rreusser
npm-org-rights scijs planeshifter
npm-org-rights scijs mikolalysenko
npm-org-rights scijs jaspervdg
npm-org-rights scijs hughsk
```
